vip-config/vip-config.php

```diff
@@ -16,6 +16,53 @@
  * @package XWP\VIP_Site_Template
  */
 
+/**
+ * Domain redirect configuration for a VIP multisite.
+ *
+ * Must run before WordPress loads to handle redirects efficiently.
+ */
```

Comment:
If this is for multisite, should we check `WP_ALLOW_MULTISITE` to start of conditional below?